src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -1019,23 +1025,23 @@ void distributable_sensitivity_computation(
                                     start_time_of_frame,
                                     end_time_of_frame,
                                     &RPC_process_related_viewgrams_sensitivity_computation,
-                                    caching_info_ptr
+                                    caching_info_ptr,
+                                    false
```

Comment:
Pass `use_KL_divergence` as false here. This is in sensitivity computation. No need to use it but distributable computation needs the bool? Could pass `NULL` instead?